style/icomoon/demo.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
```

Comment:
I think it's not so big cost to pay to have them, especially on bigger projects (which mine isn't 😆 ) because with one single click on demo.html you get to know all icons you are currently using in your project, without having to upload selection.json to icomon site for that info which is time consuming, but if you insist...